qcelemental/models/results.py

```diff
@@ -97,6 +98,107 @@ def __str__(self):
         return f"{self.__class__.__name__}({data_str})"
 
 
+class WavefunctionProperties(ProtoModel):
+
+    # The full basis set description of the quantities
+    basis: BasisSet = Schema(..., description=str(BasisSet.__doc__))
+
+    # Core Hamiltonian
+    h_core_a: Optional[Array[float]] = Schema(None, description="Alpha-spin core (one-electron) Hamiltonian.")
+    h_core_b: Optional[Array[float]] = Schema(None, description="Beta-spin core (one-electron) Hamiltonian.")
+    h_effective_a: Optional[Array[float]] = Schema(None,
+                                                   description="Alpha-spin effective core (one-electron) Hamiltonian.")
+    h_effective_b: Optional[Array[float]] = Schema(None,
+                                                   description="Beta-spin effective core (one-electron) Hamiltonian ")
+
+    # SCF Results
+    scf_orbitals_a: Optional[Array[float]] = Schema(None, description="SCF alpha-spin orbitals.")
+    scf_orbitals_b: Optional[Array[float]] = Schema(None, description="SCF beta-spin orbitals.")
+    scf_density_a: Optional[Array[float]] = Schema(None, description="SCF alpha-spin density matrix.")
+    scf_density_b: Optional[Array[float]] = Schema(None, description="SCF beta-spin density matrix.")
+    scf_fock_a: Optional[Array[float]] = Schema(None, description="SCF alpha-spin Fock matrix.")
+    scf_fock_b: Optional[Array[float]] = Schema(None, description="SCF beta-spin Fock matrix.")
+    scf_eigenvalues_a: Optional[Array[float]] = Schema(None, description="SCF alpha-spin eigenvalues.")
+    scf_eigenvalues_b: Optional[Array[float]] = Schema(None, description="SCF beta-spin eigenvalues.")
+    scf_occupations_a: Optional[Array[float]] = Schema(None, description="SCF alpha-spin occupations.")
+    scf_occupations_b: Optional[Array[float]] = Schema(None, description="SCF beta-spin occupations.")
+
```

Comment:
Add localized orbital results.

Reply:
I would like to skip this on the first pass. I am not quite sure I understand the exact requirements there.